### PR TITLE
Feature/data organization

### PR DESCRIPTION
**Pull request by itsnameissteven**
### Description
* Is this a feature or a fix?
Feature
* Where should the reviewer start?
* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?
All data for dashboard should now be viewable .  
Colors have been adjusted but not committed to them. 
There is a contrast issue with the main header. White to blue is ranking very poorly for contrast. 
Charts are able to change colors dynamically now . 
Test rewritten to reflect the data that we need back from methods. 
* What outside features or research did you use to implement this fix?
* n/a
* How should this be tested?
Review the web app.
### Relevant Screen Shots
***
#### Please review considerations below:
- [X] Follows Turing Style Guidelines
- [X] Changes generate no new warnings or errors
- [X] Checked code for and corrected any spelling errors
- [ ] README has been updated(if applicable)

### CODE REVIEW

File: src/Hydration.js [hunk 1]
```diff
@@ -13,6 +14,15 @@ class Hydration extends Calculation {
       }
     });
   }
+
+  calculateAverageWater(userRepo, id) {
+    const avg =this.calculateFirstWeekOunces(userRepo, id).reduce((acc, data) => {
```
Comment (aemiers) [inline src/Hydration.js:19]: space after = ?

**Review by aemiers — APPROVED** [review]

File: src/scripts.js [hunk 2]
```diff
@@ -118,47 +119,49 @@ function makeRandomDate(userStorage, id, dataSet) {
 
 function addHydrationInfo(id, hydrationInfo, dateString, userStorage) {
   const hydrationToday = document.getElementById('hydrationToday');
-  const hydrationAverage = document.getElementById('hydrationAverage');
   const hydroChart = document.getElementById("hydroChart");
-  chart.makeChart(hydrationInfo.calculateFirstWeekOunces(userStorage, id), hydroChart, "Number of Ounces");
-  hydrationToday.innerText = `${hydrationInfo.calculateDailyData(id, dateString, 'numOunces')}oz`;
-  hydrationAverage.innerText = `${hydrationInfo.calculateAverage(id, "numOunces")}oz`;
+  const hydrationAverageWeek = document.getElementById('hydrationAverageWeek');
+  chart.makeChart(hydrationInfo.calculateFirstWeekOunces(userStorage, id), hydroChart, "Number of Ounces", '#D5260B');
```
Comment (alexthompson207) [inline src/scripts.js:124]: Love the dynamic color parameter!

File: src/scripts.js [hunk 3]
```diff
@@ -118,47 +119,49 @@ function makeRandomDate(userStorage, id, dataSet) {
...
+  chart.makeChart(sleepInfo.calculateWeeklyData(dateString, id, userStorage, "sleepQuality"), sleepChartQuality, "Quality of Sleep", '#8B0BD5')
 }
 
 function addActivityInfo(id, activityInfo, dateString, userStorage, laterDateString, user, winnerId) {
   const stepChart = document.getElementById("stepChart");
   const flightsOfStairsChart = document.getElementById("flightsOfStairsChart");
   const activeMinutesChart = document.getElementById('activeMinutesChart');
   createBarChart(activityInfo, id, dateString, "numSteps", userStorage, stepChart, "Steps Today");
-  createBarChart(activityInfo, id, dateString, "flightsOfStairs", userStorage, flightsOfStairsChart, "flights of stairs");
+  createBarChart(activityInfo, id, dateString, "flightsOfStairs", userStorage, flightsOfStairsChart, "Flights Climbed Today");
```
Comment (alexthompson207) [inline src/scripts.js:147]: I like this name change

**Review by alexthompson207 — COMMENTED** [review]
Great job with this, the charts are such a huge addition, and they look great!

**Pull request merged**